### ISSUE TITLE
Basic english localization

### ISSUE THREAD

**Issue by winter-yuki** [braille-systems/learn-braille#88]

**Comment by zuevval** [comment]
There are many apps with english interface already in the market. But perhaps they will like our one.

**Comment by kaustika** [comment]
By basic you mean that we need to state that we do not support english yet, right?

**Comment by winter-yuki** [comment]
> By basic you mean that we need to state that we do not support english yet, right?

Yes, and maybe translate the main menu and say it in the global help

**Comment by kaustika** [comment]
> > By basic you mean that we need to state that we do not support english yet, right?
> 
> Yes, and maybe translate the main menu and say it in the global help

Okay, that's not much of a deal.
